ep/build-velox/src/get_velox.sh

```diff
@@ -16,8 +16,8 @@
 
 set -exu
 
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=2023-11-14
+VELOX_REPO=https://github.com/felixybw/velox.git
+VELOX_BRANCH=2023-11-15
```

Comment:
@FelixYBW can we use the `update` branch? this will makes it difficult make pull requests to oap/velox